source/refined-github.css

```diff
@@ -138,3 +138,13 @@ pr-branches
 .select-menu-item-text .diffstat {
 	display: inline !important;
 }
+
+/* Style the "Find feature" dialog that appears on top of the page */
+#rgh-bisect-dialog {
+	position: fixed;
+	bottom: 50%;
```

Comment:
Why move the dialog to the middle of the page?

Reply:
Feels like that'a where dialogs belong. It felt lost at the bottom. Since the rest of the page is scrollable, I think the user can either resize the window or scroll to see under it